server/src/functionalities/code-lens.ts

```diff
@@ -52,7 +41,22 @@ export const getTestCodeLenses = (file: Package): CodeLens[] => {
   ]
 }
 
-const buildRunAllTestsCodeLens = (file: Package): CodeLens =>
+export const getWollokFileCodeLenses = (file: Package): CodeLens[] =>
+  file.members.filter(isWollokDefinition).flatMap(definition => [
+    buildLens(definition, COMMAND_START_REPL, 'Run in REPL'),
+  ])
```

Comment:
Una gilada pero es mas fuerte que yo😅 

```suggestion
  file.members.filter(isWollokDefinition).map(definition =>
    buildLens(definition, COMMAND_START_REPL, 'Run in REPL'),
  )
```